MegaSolution/Sequance/UndecreasingSequance.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace Sequence
+{
+    class CheckSequence
+    {
+        const string INSTRUCTIONS1LINE = "Enter a sequance of nubers. ";
+        const string INSTRUCTIONS2LINE = "Use Space to devide numbers. Use Enter to finish input.";
+        public int[] EnterSequense ()
```

Comment:
Use separate class for sequence enter, the name of class is CheckSequence, thus it is not responsible for sequence input